### PR TITLE
ConfigFile: password encryption does not work on specific machine con…

### PR DESCRIPTION
**Pull request by haraldsteinlechner**
This pull request solves issue #1346

Note that i did not change DataProtectionScopes for decryption. This is is the minimal change i know which works (i think decrpytion ignores this). 
The docs do not say anything about this: e.g. https://msdn.microsoft.com/en-us/library/2fh8203k(v=vs.110).aspx

### CODE REVIEW

**Comment by forki** [comment]
mhm this doesn't work on travis!?

**Comment by haraldsteinlechner** [comment]
ah ok. awesome. no solution works on all configurations. We could do a simple probing - handle exception and try in local machine mode if user mode fails. what you think?

**Comment by forki** [comment]
sounds good. whatever works to make it work for more people is fine

**Comment by haraldsteinlechner** [comment]
works now

File: src/Paket.Core/ConfigFile.fs [hunk 1]
```diff
@@ -55,7 +55,12 @@ let private getRandomSalt() =
 /// Encrypts a string with a user specific keys
 let Encrypt (password : string) = 
     let salt = getRandomSalt()
-    let encryptedPassword = ProtectedData.Protect(Encoding.UTF8.GetBytes password, salt, DataProtectionScope.CurrentUser)
+    let encryptedPassword = 
+        try 
+            ProtectedData.Protect(Encoding.UTF8.GetBytes password, salt, DataProtectionScope.CurrentUser)
+        with | :? CryptographicException as e ->
+            traceWarnfn "could not protect password: %s\n for current user" e.Message
```
Comment (forki) [inline src/Paket.Core/ConfigFile.fs]: i think a verbosefn might be enough here

**Comment by forki** [comment]
thanks a lot

**Pull request merged**